game/game.py

```diff
@@ -386,25 +386,33 @@ def set_bullseye(self) -> None:
         self.blue_bullseye = Bullseye(enemy_cp.position)
         self.red_bullseye = Bullseye(player_cp.position)
 
-    def initialize_turn(self) -> None:
-        self.events = []
-        self._generate_events()
+    def initialize_turn(self, is_reinit: bool = False, for_player: bool = True) -> None:
+        if not is_reinit:
+            self.events = []
+            self._generate_events()
 
-        self.set_bullseye()
+            self.set_bullseye()
```

Comment:
Needs to be redone on reinit because base capture is a reinit. I think the only thing that should be skipped on re-init is the stats update. The bullseyes can move, the events (the front line battle location, essentially) can change, the game can end, and procurement may need to be redone.

Reply:
agree. Not thought so much on the cheat in total